docs/usage/setup.md

```diff
@@ -19,71 +30,71 @@ The Device values configuration table can be found under [General](#general) sec
 - **Value**: The valueId you want to configure
-- **Device Class**: If the value is a multilevel sensor, a binary sensor or a meter you can set a custom `device_class` to use with home assistant discovery. Check [sensor](https://www.home-assistant.io/components/sensor/#device-class) and [binary sensor](https://www.home-assistant.io/components/binary_sensor/#device-class)
+- **Device Class**: If the value is a multilevel sensor, a binary sensor, or a meter you can set a custom `device_class` to use with Home Assistant discovery. Check [sensor](https://www.home-assistant.io/components/sensor/#device-class) and [binary sensor](https://www.home-assistant.io/components/binary_sensor/#device-class)
 - **Topic**: The topic to use for this value. It is the topic added after topic prefix, node name and location. If gateway type is different than `Manual` this will be ignored
 - **QoS**: If specified, overrides MQTT settings QoS level
 - **Retain**: If specified, overrides MQTT settings retain flag
 - **Post operation**: If you want to convert your value (valid examples: '/10' '/100' '*10' '*100')
 - **Parse send**: Enable this to allow users to specify a custom `function(value,valueId,node,logger)` to parse the value sent to MQTT. The function must be sync
 - **Parse receive**: Enable this to allow users to specify a custom `function(value,valueId,node,logger)` to parse the value received via MQTT. The function must be sync
-- **Enable Poll**: Enable poll of this value by using zwave-js [pollValue](https://zwave-js.github.io/node-zwave-js/#/api/node?id=pollvalue)
+- **Enable Poll**: Enable poll of this value by using Z-Wave JS [pollValue](https://zwave-js.github.io/node-zwave-js/#/api/node?id=pollvalue)
 - **Poll interval**: Seconds between two poll requests
 
-## Zwave
+## Z-Wave
 
 - **Serial port**: The serial port where your controller is connected
-- **Security Keys** : Zwave network keys for secure inclusion S0/S2. The correct format is like the OZW key but without `0x` `,` and spaces: OZW: `0x5C, 0x14, 0x89, 0x74, 0x67, 0xC4, 0x25, 0x98, 0x51, 0x8A, 0xF1, 0x55, 0xDE, 0x6C, 0xCE, 0xA8` Zwavejs: `5C14897467C42598518AF155DE6CCEA8`
+- **Security Keys** : Z-Wave network keys for secure inclusion S0/S2. The correct format is like the OZW key but without `0x` `,` and spaces: OZW: `0x5C, 0x14, 0x89, 0x74, 0x67, 0xC4, 0x25, 0x98, 0x51, 0x8A, 0xF1, 0x55, 0xDE, 0x6C, 0xCE, 0xA8` Zwavejs: `5C14897467C42598518AF155DE6CCEA8`
   - **S2 Unauthenticated**: Like S2 Authenticated, but without verification that the correct device is included (skip DSK verification step)
   - **S2 Authenticated**: Security systems, sensors, lighting, etc.
   - **S2 AccessControl** (highest): Used for Door locks, garage doors, etc.
   - **S0 Legacy** (lowest): The previous **networkKey** setting, used for S0 security
```

Comment:
Is there any reason, those are sorted from medium to highest, followed by lowest? :)

Reply:
nope 😆

Reply:
Is it still necessary to reference the OZW format? I think it only adds confusion at this point in time.

Reply:
There are still users asking me how to correctly set it so I would keep it there for now

Reply:
It would be easier if you could just paste the old key in there and it gets formatted correctly.
![keys](https://user-images.githubusercontent.com/17641229/132916302-4b8cf65e-9f66-4e24-8111-684ae32e6aee.gif)

Reply:
I like the idea of being able to auto format it. I'll open a feature request.